### PR TITLE
New PR with the codemeta.json

### PR DESCRIPTION
**Pull request by bkhelifi**
Signed-off-by: Bruno Khelifi <khelifi@in2p3.fr>

**Description**
This PR replaces the PR #4078 due to merge conflicts, that should not be merged.

### CODE REVIEW

File: codemeta.json [hunk 1]
```diff
@@ -1,703 +1,372 @@
...
+        "affiliation": {
+            "@type": "Organization",
+            "name": "Center for Astrophysics | Harvard & Smithsonian, CfA, 60 Garden St., 02138 Cambridge MA, USA"
+        },
+        "familyName": "Donath",
+        "givenName": "Axel"
+    },
+    "developmentStatus": "active",
+    "issueTracker": "https://github.com/gammapy/gammapy/issues",
+    "funding": "None, apart for the permanent staff",
```
Comment (registerrier) [inline codemeta.json:42]: and not the non-permanent?
How is it described by others?

File: codemeta.json [hunk 2]
```diff
@@ -1,703 +1,372 @@
...
+        "numpy>=1.20",
+        "scipy>=1.4",
+        "astropy>=5.0",
+        "regions>=0.5.0",
+        "pyyaml>=5.1",
+        "click>=7.0",
+        "pydantic>=1.4",
+        "iminuit>=2.8.0",
+        "matplotlib>=3.4",
+        "naima",
```
Comment (registerrier) [inline codemeta.json:65]: Optional dependencies might be left out no? There are not requirements, per se.

File: codemeta.json [hunk 3]
```diff
@@ -1,703 +1,372 @@
...
         {
-            "@id": "https://orcid.org/0000-0002-4198-4005",
             "@type": "Person",
-            "affiliation": {
-                "@type": "Organization",
-                "name": "HeidelbergCement"
-            },
-            "familyName": "Deil",
-            "givenName": "Christoph"
+            "givenName": "Jos\u00e9 Vin\u00edcius",
```
Comment (registerrier) [inline codemeta.json:129]: Are those characters expected?

File: codemeta.json [hunk 4]
```diff
@@ -1,703 +1,372 @@
...
-        {
-            "@id": "https://orcid.org/0000-0002-5877-9438",
-            "@type": "Person",
-            "affiliation": {
-                "@type": "Organization",
-                "name": "AstroParticule et Cosmologie, Universit\u00e9 de Paris Cit\u00e9, CNRS"
-            },
-            "familyName": "Hajlaoui",
-            "givenName": "Jalel Eddine"
+            "affiliation": "Universit\u00e9 Paris Cit\u00e9, CNRS, Astroparticule et Cosmologie, F-75013 Paris, France",
```
Comment (registerrier) [inline codemeta.json:150]: special characters as expected?

**Review by registerrier — COMMENTED** [review]
Thanks @bkhelifi .

Some small comments inline.

File: codemeta.json [hunk 5]
```diff
@@ -1,703 +1,372 @@
 {
     "@context": "https://doi.org/10.5063/schema/codemeta-2.0",
     "@type": "SoftwareSourceCode",
+    "codeRepository": "https://github.com/gammapy/gammapy",
+    "description": "Gammapy analyzes gamma-ray data and creates sky images, spectra and lightcurves, from event lists and instrument response information; it can also determine the position, morphology and spectra of gamma-ray sources. It is used to analyze data from H.E.S.S., Fermi-LAT, HAWC, and the Cherenkov Telescope Array (CTA).",
```
Comment (AtreyeeS) [inline codemeta.json:5]: not MAGIC?

File: codemeta.json (hunk 1 above)
Comment (bkhelifi) [reply codemeta.json:42]: May I propose, "None, apart from one-off funding from the CC institutes"?

File: codemeta.json (hunk 3 above)
Comment (bkhelifi) [reply codemeta.json:129]: UTF8 and Jason... To be honest, I do not know.

File: codemeta.json (hunk 5 above)
Comment (bkhelifi) [reply codemeta.json:5]: Do we have a science paper with MAGIC data?

File: codemeta.json (hunk 4 above)
Comment (bkhelifi) [reply codemeta.json]: I have updated the file to fix that

File: codemeta.json (hunk 5 above)
Comment (AtreyeeS) [reply codemeta.json:5]: Ah probably not! @cosimoNigro ?

Comment (bkhelifi) [reply codemeta.json:5]: I put the names of experiments for which Science paper have been published using Gammapy. And I have not found a paper wtith a an analysis with Gammapy of MAGIC or VERITAS data. They are papers that analyse Fermi-LAT data with Gammapy

File: codemeta.json (hunk 2 above)
Comment (registerrier) [reply codemeta.json:65]: Well if we take the description it says `Required software dependencies` so I think only required dependencies should go in.

File: codemeta.json (hunk 1 above)
Comment (registerrier) [reply codemeta.json:42]: Maybe drop the `funding` keyword then.

File: codemeta.json [hunk 6]
```diff
@@ -1,703 +1,372 @@
...
+        "astropy>=5.0",
+        "regions>=0.5.0",
+        "pyyaml>=5.1",
+        "click>=7.0",
+        "pydantic>=1.4",
+        "iminuit>=2.8.0",
+        "matplotlib>=3.4",
+        "naima",
+        "sherpa",
+        "healpy",
```
Comment (registerrier) [inline codemeta.json:67]:
```suggestion
```

File: codemeta.json [hunk 7]
```diff
@@ -1,703 +1,372 @@
...
+        "requests",
+        "tqdm",
+        "ipywidgets"
```
Comment (registerrier) [inline codemeta.json:70]:
```suggestion
        "naima",
        "sherpa",
        "healpy",
        "requests",
        "tqdm"
```

File: codemeta.json [hunk 8]
```diff
@@ -1,703 +1,369 @@
...
+        "astropy>=5.0",
+        "regions>=0.5.0",
+        "pyyaml>=5.1",
+        "click>=7.0",
+        "pydantic>=1.4",
+        "iminuit>=2.8.0",
+        "matplotlib>=3.4",
+        "requests",
+        "tqdm",
+        "ipywidgets"
```
Comment (registerrier) [inline codemeta.json:67]:
```suggestion
```

File: codemeta.json (hunk 1 above)
Comment (registerrier) [reply codemeta.json:42]: Indeed in practice one `funding` goes with a `funder`.

File: codemeta.json [hunk 9]
```diff
@@ -1,703 +1,368 @@
...
+        "affiliation": {
+            "@type": "Organization",
+            "name": "Center for Astrophysics | Harvard & Smithsonian, CfA, 60 Garden St., 02138 Cambridge MA, USA"
+        },
+        "familyName": "Donath",
+        "givenName": "Axel"
+    },
+    "developmentStatus": "active",
+    "issueTracker": "https://github.com/gammapy/gammapy/issues",
+    "funding": "None, apart for the permanent staff",
```
Comment (registerrier) [inline codemeta.json:42]:
```suggestion
```

**Pull request merged**